test/e2e/virtual/apiexport/authorizer_test.go

```diff
@@ -509,8 +506,10 @@ func TestAPIExportBindingAuthorizer(t *testing.T) {
 
 	orgPath, _ := framework.NewOrganizationFixture(t, server) //nolint:staticcheck
 
-	serviceProviderPath, _ := kcptesting.NewWorkspaceFixture(t, server, orgPath, kcptesting.WithName("service-provider"))
-	tenantPath, tenantWorkspace := kcptesting.NewWorkspaceFixture(t, server, orgPath, kcptesting.WithName("tenant"))
+	// IMPORTANT: This does not test sharded setup.
```

Comment:
what does it mean?

Reply:
This test uses only the root shard. To test the same test across shards, we must create fake tenants on each shard, bound to the main API export to trigger the URL population flow on the API export endpoints.

I can see how much it would take to convert this to a shared test.

Reply:
updated.